jolt-core/src/host/toolchain.rs

```diff
@@ -130,23 +130,72 @@ fn unpack_toolchain(channel: &str) -> Result<()> {
 async fn download_toolchain(client: &Client, url: &str) -> Result<()> {
     let jolt_dir = jolt_dir();
     let output_path = jolt_dir.join("rust-toolchain.tar.gz");
+    println!("Downloading toolchain to {output_path:?}");
```

Comment:
We've decided to use `info!`, `debug!`, `warn!`, and `error!` macros from `tracing` crate.
Let's just use `info!` here.